docs/install_guide.adoc

```diff
@@ -0,0 +1,180 @@
+= User Installation Guide
+
+== Getting Started
+
+You will need an operational operator deployed OpenStack system
+with the Telemetry operator installed. The following links point
+to documents detailed how to create this required starting environment:
+
+* https://github.com/openstack-k8s-operators/openstack-operator[Openstack Operator]
+* https://github.com/openstack-k8s-operators/telemetry-operator[Telemetry Operator]
+* https://kubernetes.io/docs/concepts/extend-kubernetes/operator/[Kubernetes operators]
+* https://prometheus.io/[Prometheus metrics]
+
+To verify that the environment set up is ready,
+
+Log in to the Kubernetes/Openshift environment
+Example uses CRC (https://crc.dev/docs/introducing/[Code Ready Containers]):
```

Comment:
there is nothing in the commands below that make this 'crc specific'  ... 
it may be worth moving this up into the Getting Started section to make the point that 'crc installation is adequade for a developer environment'